package.json

```diff
@@ -18,7 +18,6 @@
     "watch:css": "autoless --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
     "test": "npm run test:lint",
     "test:lint": "eslint --ext .js,.jsx src/",
-    "test:unit": "mocha -R spec ./test/unit/*.js",
```

Comment:
This must have been copied over from another manifest. There aren't any tests here...